diff --git a/backend/app/main.py b/backend/app/main.py
index 4f13bae095871194868b17e709f02b8676ae68de..a1ae23269a1fbeadf594e94ac7dcf443d617bb57 100644
--- a/backend/app/main.py
+++ b/backend/app/main.py
@@ -1,28 +1,28 @@
 from fastapi import FastAPI, HTTPException
 from pydantic import BaseModel
-from ai_model import predict_persona
+from .ai_model import predict_persona
 from fastapi.middleware.cors import CORSMiddleware
 import pandas as pd
 from collections import Counter
 from datetime import datetime
 import requests
 from fastapi import Request
 app = FastAPI()
 import os
 from dotenv import load_dotenv
 from openai import OpenAI
 load_dotenv()
 client = OpenAI(
     base_url="https://openrouter.ai/api/v1",
     api_key=os.getenv("HF_API_KEY")
 )
 app.add_middleware(
     CORSMiddleware,
     allow_origins=["http://localhost:5173"],  
     allow_credentials=True,
     allow_methods=["*"],
     allow_headers=["*"],
 )
 df = pd.read_csv("../app/data/shoplens_temiz_veri_cleaned.csv", sep=";",encoding="utf-8")
 users_df = pd.read_csv("../app/data/All_Data_MailPassword_withRoles.csv", sep=";",encoding="utf-8")
 user_dict = dict(zip(users_df["email"], users_df["password"]))
diff --git a/backend/requirements.txt b/backend/requirements.txt
index 97dc7cd8c1fd2f07d8ec79a1117664c5ebaf2842..5455cdbe6563b6cadeab05869859bd80415ffc35 100644
--- a/backend/requirements.txt
+++ b/backend/requirements.txt
@@ -1,2 +1,8 @@
+pandas
 fastapi
 uvicorn
+joblib
+scikit-learn
+python-dotenv
+openai
+requests
